src/commands/endpoint/list.ts

```diff
@@ -0,0 +1,33 @@
+import { Command } from "@oclif/core";
+import { ShellConfig } from "../../lib/config";
+import chalk from "chalk";
+
+export default class ListEndpointCommand extends Command {
+  static flags = {};
+
+  static description = "Lists endpoints in ~/.fauna-shell.";
```

Comment:
Looks like we typically don't use present tense in the descriptions, except for cloud-login. Please align.

Reply:
We bounce back and forth between imperative and indicative mood. The schema commands and import, e.g., use imperative, but lots of others use indicative, like the delete-* and create-* commands. I prefer imperative: "Compute the frustum of the wangledinger", not "Computes the frustum of the wangledinger".

Reply:
speaking of `delete-*`, do we want `fauna endpoint delete` or `fauna endpoint remove`? I added called it remove and noticed it was called delete beforehand. I'd be fine with either, with a very slight preference to `remove` (simply because all you're doing is removing a value from a config - you're not deleting any data).

Reply:
we discussed in standup, and we want to go with `remove`. thats what is currently implemented, so leaving as-is.